torch/testing/_internal/common_methods_invocations.py

```diff
@@ -419,6 +419,52 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad):
+    test_sizes = [
```

Comment:
These generated inputs seem very thorough, which is great, does it make the grad and/or gradgradchecks take a long time to run, however?

Reply:
You're right, this does take a very long time. I just realized that the return statement below is indented too far, so it was only adding the first test size. After fixing that, the command `python test/test_ops.py -k vector_norm` takes more than 20 minutes to complete on my machine. I'll prune down some of the test cases

Reply:
After making some changes, `$ python test/test_ops.py -k vector_norm` takes about 70 seconds to run for me. Is this alright, or would it be better to prune it further?

Reply:
70s is a long time. Are there any tests that take a particular long time? Always using (2, 2) for the 2D size sounds good. Adding an empty tensor input may be OK, too. If we want to prune further maybe the 3.5 or 2 ord can go?

Reply:
@ngimel and I recommend removing 2 and 3.5 for now. Also always using (2, 2) instead of (S, S)

Reply:
After my last update, `python test/test_ops.py -k vector_norm` now takes 35 seconds. If that's still too long, we could consider not testing both `keepdim=True` and `keepdim=False` for all cases